prow/config/config.go

```diff
@@ -1919,6 +1940,29 @@ func ValidateController(c *Controller) error {
 	return nil
 }
 
+func ValidateReportTemplates(c *Controller) error {
+	if len(c.ReportTemplateString) > 0 {
+		if len(c.ReportTemplateStrings) > 0 {
+			return errors.New("both report_template and report_templates are specified")
+		}
+
+		logrus.Warning("report_template will be deprecated on September 2020, and it will be replaced with report_templates['*']")
+		c.ReportTemplateStrings = make(map[string]string)
+		c.ReportTemplateStrings["*"] = c.ReportTemplateString
+	}
+
+	c.ReportTemplates = make(map[string]*template.Template)
+	for orgRepo, value := range c.ReportTemplateStrings {
+		reportTmpl, err := template.New("Report").Parse(value)
+		if err != nil {
+			return fmt.Errorf("parsing template: %v", err)
```

Comment:
Please make the life of ppl operating Prow better by adding the key to the error